api/Document.json

```
@@ -8228,30 +8228,72 @@
           }
         }
       },
-      "policy": {
+      "featurePolicy": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/policy",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/featurePolicy",
           "support": {
-            "chrome": {
-              "version_added": "69",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#enable-experimental-productivity-features",
-                  "value_to_set": "Enabled"
-                }
-              ]
-            },
-            "chrome_android": {
-              "version_added": "69",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#enable-experimental-productivity-features",
-                  "value_to_set": "Enabled"
-                }
-              ]
-            },
+            "chrome": [
+              {
+                "version_added": "74",
+                "notes": "<a href='https://www.chromestatus.com/feature/5190687460950016'>Chrome Platform Status</a> incorrectly describes it as <code>document.policy</code>. It was renamed in <a href='https://crbug.com/917057'>bug 917057</a> before being shipped in <a href='https://crbug.com/703703#c22'>bug 703703</a>."
```

We can probably improve the wording on this.
```suggestion
                "notes": "<a href='https://www.chromestatus.com/feature/5190687460950016'>Chrome Platform Status</a> incorrectly describes this variable as <code>document.policy</code>. It was renamed in <a href='https://crbug.com/917057'>bug 917057</a> before being shipped in <a href='https://crbug.com/703703#c22'>bug 703703</a>."
```